File: Module/PlayWays/src/main/java-room-grab/com/module/playways/grab/room/songmanager/model/ChangeTagSuccessEvent.java
package com.module.playways.grab.room.songmanager.model;

import com.component.busilib.friends.SpecialModel;

public class ChangeTagSuccessEvent {
    SpecialModel specialModel;

    public SpecialModel getSpecialModel() {
        return specialModel;
    }

    public ChangeTagSuccessEvent(SpecialModel specialModel) {
        this.specialModel = specialModel;
    }
}

File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/event/ChangeTagSuccessEvent.kt
package com.module.playways.songmanager.event

import com.component.busilib.friends.SpecialModel

class ChangeTagSuccessEvent(specialModel: SpecialModel) {
    var specialModel: SpecialModel
        internal set

    init {
        this.specialModel = specialModel
    }
}
